tests/common/config/unit/test_config.py

```diff
@@ -0,0 +1,144 @@
+import unittest
+
+import os
+
+import ims.common.config as config
+import ims.exception.config_exceptions as config_exceptions
+from ims.common.log import trace
+
+
+class TestBadPath(unittest.TestCase):
+    """ Tests whether an exception is raised when loading from invalid path """
+
+    @trace
+    def setUp(self):
+        self.old_path = os.getenv('BMI_CONFIG')
+        os.environ['BMI_CONFIG'] = "blahblah.cfg"
+
+    def runTest(self):
+        with self.assertRaises(IOError):
+            config.load(force=True)
+
+    def tearDown(self):
+        os.environ['BMI_CONFIG'] = self.old_path
+        config.load(force=True)
+
+
+class TestCorrectPath(unittest.TestCase):
+    """ Tests if able to load from correct path """
+
+    @trace
+    def setUp(self):
+        self.old_path = os.getenv('BMI_CONFIG')
+        os.environ['BMI_CONFIG'] = "tests/common/config/config1.cfg"
+
+    def runTest(self):
+        config.load(force=True)
+        cfg = config.get()
+        self.assertEqual(cfg.tests.var1, '0')
```

Comment:
quick question: our config file has to have a `[test]` section with a variable called `var1` just for this test? If that's the case, perhaps we could give it a more descriptive name; but even I can't think of something more appropriate right now.